migrations/1597654308406-remove-importHashes-from-users-with-accounts.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
```

Comment:
Codacy found an issue: ['mongoose' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=6024362)